controllers/operatorconfigmap_controller.go

```diff
@@ -279,36 +289,29 @@ func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	// ownerReference on it as its cluster scoped resource
 	cephfsCSIDriver := templates.CephFSCSIDriver.DeepCopy()
 	cephfsCSIDriver.ObjectMeta.Name = csi.GetCephFSDriverName()
-	err = csi.CreateCSIDriver(c.ctx, c.Client, cephfsCSIDriver)
-	if err != nil {
+	if err := csi.CreateCSIDriver(c.ctx, c.Client, cephfsCSIDriver); err != nil {
```

Comment:
I don't think there's need for cosmetic changes as we are returning on every non-nil value, changing from `err = func() bool; if err != nil {}` to `if err := func() bool; err !=nil {}` is kinda adding extra diff.

Reply:
@leelavg moving any set of variables into a deeper scope (if it is only needed there) is a welcomed change.